plugins/GSdx/GSUtil.cpp

```diff
@@ -396,6 +396,39 @@ D3D_FEATURE_LEVEL GSUtil::CheckDirect3D11Level(IDXGIAdapter *adapter, D3D_DRIVER
 	return SUCCEEDED(hr) ? level : (D3D_FEATURE_LEVEL)0;
 }
 
+GSRendererType GSUtil::GetBestRenderer()
+{
+	CComPtr<IDXGIAdapter1> adapter;
+	DXGI_ADAPTER_DESC1 desc;
+	CComPtr<IDXGIFactory1> DXGI;
+
+	if (GSUtil::CheckDXGI())
+	{
+		CreateDXGIFactory1(__uuidof(IDXGIFactory1), (void**)&DXGI);
+	}
+
+	if(GSUtil::CheckD3D11())
+	{
+		DXGI->EnumAdapters1(NULL, &adapter);
```

Comment:
Because people usually use `NULL` for pointers (C++ before nullptr was introduced, and also C) and will associate it with pointers.